examples/analytics-vendors.amp.html

```diff
@@ -715,6 +715,23 @@
 </amp-analytics>
 <!-- End New Relic example -->
 
+<!--Catchpoint example--> 
```

Comment:
Hello,
Apologize for the multiple commits.
I have updated the branch accordingly, let me know if the test stub work for the vendor request.
Yes, please encoded the query string.